clang/include/clang/Driver/Options.td

```diff
@@ -6919,6 +6919,10 @@ defm loop_versioning : BoolOptionWithoutMarshalling<"f", "version-loops-for-stri
 
 def fhermetic_module_files : Flag<["-"], "fhermetic-module-files">, Group<f_Group>,
   HelpText<"Emit hermetic module files (no nested USE association)">;
+
+def do_concurrent_to_openmp_EQ : Joined<["-"], "fdo-concurrent-to-openmp=">,
```

Comment:
```suggestion
def fdo_concurrent_to_openmp_EQ : Joined<["-"], "fdo-concurrent-to-openmp=">,
```

Other `f` flags seem to carry the  f name.

Reply:
In the case above I was only suggesting the internal name of the flag also to carry the `f` prefix. Otherwise the user visible part of the flag already had the `f` prefix. 

Generally, machine independent feature flags have the `f` prefix.